docs/evm/Build/vm-bridge.md

```diff
@@ -0,0 +1,11 @@
+---
+title: VM Bridge
+sidebar_label: VM Bridge
+sidebar_position: 1
+---
+
+# VM Bridge
+
+Flow maintains a VM bridge, which enables Cadence to interact with FlowEVM. The entrypoint to FlowEVM is always through this VM bridge. The VM bridge is responsible for creating cadence owned accounts on FlowEVM and their corresponding resources on Cadence (read more on cadence owned accounts [here](./accounts.md)). The VM bridge is also responsible for moving fungible tokens and non-fungible tokens between Cadence and FlowEVM. 
```

Comment:
```suggestion
Flow maintains a VM bridge, which enables Cadence to interact with FlowEVM. The entrypoint to FlowEVM is always through this VM bridge. The VM bridge is responsible for creating cadence owned accounts on FlowEVM and their corresponding resources on Cadence (read more on cadence owned accounts [here](./accounts.md)). The VM bridge is also responsible for moving fungible tokens and non-fungible tokens between Cadence and FlowEVM. 
```

This needs changing since the reference to VM bridge creating account is incorrect and we could add a bunch more detail. However, it depends on whether this meets MVP scope